### DISCUSSION TITLE
Auth token seems to not work in python script

### DISCUSSION THREAD

**Discussion by Shrezno** [streamlink/streamlink#5874] [Q&A] [answered]
### Description

I have a recorder hooked up to a discord bot I made and the recording and the bot itself seems to work but the recordings have ads burned in.

I have twitch turbo and when I use streamlink in a cmd script there are no burned in ads
I'm on streamlink 6.6.2. and I checked that my AUTH is up to date
I tried a couple different variations but most recent one was this 
`streamlink.set_option('http-header', 'Authorization=OAuth {key.OAUTH}')` 
and i have a key.py with `OAUTH = 'xxxxxxxxxxxxxxxxxxx'`

When I'm using python and I record channels I'm not subbed to I get pre-rolls and if I record channels I am subbed to I get mid-rolls

**Comment by bastimeyer** [comment, marked as answer]
> `streamlink.set_option('http-header', 'Authorization=OAuth {key.OAUTH}')`

1. The value you're trying to set is not an f-string
2. `http-header` is not a valid session option. It's `http-headers`. Session options don't 1:1 map to CLI arguments. See the API docs:
   https://streamlink.github.io/api/session.html#streamlink.session.options.StreamlinkOptions

Setting Twitch auth headers doesn't guarantee that you won't get embedded ads.
